tests/lib/network.sh

```diff
@@ -3,3 +3,16 @@
 get_default_iface(){
     ip route get 8.8.8.8 | awk '{ print $5; exit }'
 }
+
+wait_listen_port(){
+    PORT="$1"
+
+    for _ in $(seq 120); do
+        if ss -lnt | grep -Pq "LISTEN.*?:$PORT +.*?\n*"; then
```

Comment:
The output of `ss -lnt` has the following format (sampled form my desktop):

```
zyga@fyke2:~$ ss -lnt
State Recv-Q Send-Q                       Local Address:Port  Peer Address:Port 
LISTEN0      5                              10.247.47.1:53         0.0.0.0:*    
LISTEN0      128                          127.0.0.53%lo:53         0.0.0.0:*    
LISTEN0      128                                0.0.0.0:22         0.0.0.0:*    
LISTEN0      5                                127.0.0.1:631        0.0.0.0:*    
LISTEN0      128                              127.0.0.1:6010       0.0.0.0:*    
LISTEN0      5                 [fd42:5407:206a:ac7b::1]:53            [::]:*    
LISTEN0      5       [fe80::d866:34ff:fee8:e644]%lxdbr0:53            [::]:*    
LISTEN0      128                                   [::]:22            [::]:*    
LISTEN0      5                                    [::1]:631           [::]:*    
LISTEN0      128                                  [::1]:6010          [::]:*    
```

As such the regular expression looks okay.